src/test/java/domain/AttendanceTest.java

```diff
@@ -0,0 +1,69 @@
+package domain;
+
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+import static org.junit.jupiter.api.Assertions.*;
+import static util.Constants.ERROR_HEADER;
+
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.time.LocalTime;
+import org.assertj.core.api.ThrowableAssert.ThrowingCallable;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import view.InputValidator;
+
+public class AttendanceTest {
+    LocalDate validDate = LocalDate.of(2024, 12, 3);
+    LocalTime validTime = LocalTime.of(10, 0);
+
+    @DisplayName("출석 날짜와 시간을 입력하여 출석 기록을 저장한다.")
+    @Test
+    void test1() {
+        assertDoesNotThrow(() -> new Attendance(
+                LocalDateTime.of(validDate, validTime)));
+
+    }
+
+    @DisplayName("캠퍼스 운영시간 전이면 예외가 발생한다.")
+    @Test
+    void test2() {
+        LocalTime beforeRunningTime = LocalTime.of(7, 59);
+
+        assertThrowsIllegalArgumentException(
+                () -> new Attendance(validDate, beforeRunningTime));
+    }
```

Comment:
```suggestion
    @ParameterizedTest(name = "캠퍼스 운영시간인 {0}보다 이전이면 예외가 발생한다.")
    @MethodSource("provideBeforeRunningTime")
    void test2(LocalTime runningTime) {
        var beforeRunningTime = runningTime.minusMinutes(10);
        assertThrowsIllegalArgumentException(
                () -> new Attendance(validDate, beforeRunningTime));
    }
```
<img width="432" alt="Screenshot 2025-03-03 at 11 35 01" src="https://github.com/user-attachments/assets/35111bba-95b6-4838-b65a-b53de340bef7" />

이런식으로 된다면 나중에 운영시간이 변경된다고 해도 운영시간만 바꿔주면 모든 테스트가 다 통과할 수 있지 않을까요?